x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/lang_ident/lang_ident.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import React, { FC, useState, useMemo } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EuiSpacer, EuiTextArea, EuiButton, EuiTabs, EuiTab } from '@elastic/eui';
+
+import { useMlApiContext } from '../../../../../contexts/kibana';
+import { LangIdentInference } from './lang_ident_inference';
+import type { FormattedLangIdentResp } from './lang_ident_inference';
+import { LangIdentOutput } from './lang_ident_output';
+import { MLJobEditor } from '../../../../../jobs/jobs_list/components/ml_job_editor';
+import { extractErrorMessage } from '../../../../../../../common/util/errors';
+import { ErrorMessage } from '../../inference_error';
+import { OutputLoadingContent } from '../../output_loading';
+
+interface Props {
+  model: estypes.MlTrainedModelConfig;
+}
+
+enum TAB {
+  TEXT,
+  RAW,
+}
+
+export const LangIdentModel: FC<Props> = ({ model }) => {
+  const { trainedModels } = useMlApiContext();
+
+  const ner = useMemo(() => new LangIdentInference(trainedModels, model), [trainedModels, model]);
+
+  // const [inputText, setInputText] = useState('');
```

Comment:
commented code

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/2a9f47840a7e077a838e19e46673fad819b0f80c